src/elements/a-event/index.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  'a-click': require('./a-click'),
-  'a-event': require('./a-event'),
-  'a-hover': require('./a-hover'),
-  'a-load': require('./a-load'),
-  'a-mousedown': require('./a-mousedown'),
```

Comment:
that breaks _a lot_ of people's sites. can we have breaking changes happen in `1.0.0` or the next release?

at least make a deprecation if you intend to.